templates/Phinx/create.php.template

```diff
@@ -6,12 +6,12 @@ use $useClassName;
 class $className extends $baseClassName
 {
     /**
-     * Change Method.
-     *
-     * More information on this method is available here:
-     * http://docs.phinx.org/en/latest/migrations.html#the-change-method
-     * @return void
-     */
```

Comment:
The previous indentation of one extra space is correct for our style guide. The new doc blocks you added are inconsistent with the rest of this project. Could we get them updated?

Reply:
I will fix this